patches/mesa-issue-2034.patch

```diff
@@ -0,0 +1,10 @@
+--- a/mesa-src/src/gallium/state_trackers/osmesa/osmesa.c
```

Comment:
Please add these files to https://github.com/servo/osmesa-src/blob/6ba41c7d227be8cb745a176b53ae39fbeec99f0a/tools/update-mesa.sh#L35.

Reply:
Done.